plugins/abnormal_security/help.md

```diff
@@ -221,6 +299,7 @@ _This plugin does not contain any troubleshooting information._
 
 # Version History
 
+* 1.1.0 - New action Get Cases | New action Get Case Details
```

Comment:
```suggestion
* 1.1.0 - New actions Get Cases and Get Case Details
```

Reply:
This follows style specified in https://docs.rapid7.com/insightconnect/style-guide